spec/acceptance/destkeypass_spec.rb

```diff
@@ -36,7 +36,7 @@
 
   it 'cannot make a cert req with the wrong password' do
     shell("\"#{@keytool_path}keytool\" -certreq -alias broker.example.com -v "\
-     "-keystore #{target} -storepass testpass -keypass qwert",
+     "-keystore #{target} -storepass qwert -keypass qwert",
```

Comment:
Can you confirm why this needed to be changed?

Reply:
When on ubuntu 18.04 with the new java version storepass and keypass need to be the same